tests/TestLaunchExporter.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import os
+import unittest
+
+sys.path.append('.')
+from unittest import TestCase
+from vrops_exporter import parse_params
+
+
+class TestLaunchExporter(TestCase):
+
+    def test_with_cli_params_1(self):
+        sys.argv = ['prog', '-u', 'testuser', '-p', 'testpw31!', '-o', '1234']
+        parse_params()
+        self.assertEqual(os.getenv('USER'), 'testuser', 'The user was not set correctly!')
+        self.assertEqual(os.getenv('PASSWORD'), 'testpw31!', 'The password was not set correctly!')
+        self.assertEqual(os.getenv('PORT'), '1234', 'The port was not set correctly!')
+
+    def test_with_cli_params_2(self):
+        sys.argv = ['prog', '--user', 'testuser', '--password', 'testpw31!', '--port', '1234']
+        parse_params()
+        self.assertEqual(os.getenv('USER'), 'testuser', 'The user was not set correctly!')
+        self.assertEqual(os.getenv('PASSWORD'), 'testpw31!', 'The password was not set correctly!')
+        self.assertEqual(os.getenv('PORT'), '1234', 'The port was not set correctly!')
+
+    def test_with_cli_and_env_params(self):
```

Comment:
maybe slightly different naming:
cli params like: `cli_testuser`
env params like: `env_testuser` 
so they can be differentiated easier without specifically looking which one was defined where